lib/home_page.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/features/pokemon_favorites/favorite_connector.dart';
+import 'package:pokedex/features/pokemon_overview/pokemon_list_connector.dart';
+import 'package:pokedex/utils/strings.dart' as str;
+
+class HomePage extends StatelessWidget {
+  const HomePage({
+    Key? key,
+    required this.child,
+  }) : super(key: key);
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    final router = GoRouter.of(context);
```

Comment:
kindly have this properties private 🙂

-  router
- currentNavIndex
-  navBarItems